peps/pep-0772.rst

```diff
@@ -0,0 +1,463 @@
+Carl Meyer and Jannis Leidel. Over the years, additional projects got added to
+the PyPA and the tooling maintained by the PyPA became the baseline tooling for
+Python packaging, with additional package ecosystems using and building on top
+of the tools and interoperability standards maintained by the PyPA.
+
+:pep:`609` formalised PyPA's authority over the existing packaging tools, the
+interoperability standards and their operation as a group of independent
+projects under the same umbrella that aim to be interoperable. It also stated
+that PyPA should be expected to provide opinions, insight and experience when
+ecosystem-wide changes are being proposed. In particular, the work to support
+existing projects and maintain a user guide has been excellent.
+
+At the time of writing, PyPA is still less formal than its counterpart on the
+CPython side. It does not have a community elected body and individuals within
+the group meet ad hoc, rather than on a regular cadence. The PyPA itself is
+defined by a group of projects rather than as a group of individuals, with no
+direct oversight over its individuals.
+
+------------
+Packaging-WG
+------------
+
+The PSF's `Packaging Workgroup`_ was created to support the larger efforts of
+improving and maintaining the packaging ecosystem in Python through fundraising
+and disbursement of raised funds. It was expected to largely focus on efforts
+such as PyPI, pip, packaging.python.org, Setuptools, and cross-project efforts.
+
+The group made very good progress here in the past, but the scope and scale of
+supporting Python's packaging ecosystem has grown considerably in the last few
+years. There is both more work to be done and more stakeholders than ever
+before. The Packaging Workgroup is also not a community elected body and does
+not have a regular mechanism for changing, adding or removing members.
+
+--------------------------
+Interoperability Standards
+--------------------------
+
+The PyPA is responsible for creating and maintaining the interoperability
+standards for Python packaging tools. The decision making for updating these
+standards involves a standing delegation from the Python Steering Council to
+specific individuals based on specific topics, and these individuals have the
+ability to further delegate the decision making on the changes to other
+individuals.
+
+We know that this process is not sustainable and there has been a desire
+expressed by many to have a designated body to make these decisions (including
+the individuals with the standing delegations).
+
+-----------------------
+Python Steering Council
+-----------------------
+
+While there is overlap between the PyPA, Packaging-WG as well as the Python
+core team, the Steering Council is not well-placed to make decisions over
+Python packaging matters directly. Packaging only tangentially intersects with
+Python language evolution, packaging requires specialised expertise and deep
```

Comment:
There's a part of me that wants the time machine to go back and make a very clear distinction between import packages and distribution packages.
Python's import package behavior is very much tied to the language's evolution, is it not?

Would it make sense to clarify the use of the term "packaging" early on to refer to distribution packaging in the context of this PEP to stave off further confusion?
Unless the intent is that the Packaging Council will actively work to evolve the Python language's changes to imports?